keyboards/crkbd/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,269 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
+#include "keymap_french_osx.h"
+#include "keymap_bepo.h"
+#include "sendstring_bepo.h"
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BEPO  0
```

Comment:
Lines 27-30 should be replaced with:

```c
enum layer_names {
    _BEPO,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

Non-consecutive layer indexes will increase your firmware size for no benefit.